malariagen_data/anoph/h12.py

```diff
@@ -514,6 +514,269 @@ def plot_h12_gwss(
+
+        # Tidy up the plot.
+        fig.yaxis.axis_label = "H12"
+        fig.yaxis.ticker = [0, 1]
+        self._bokeh_style_genome_xaxis(fig, contig)
+
+        if show:  # pragma: no cover
+            bokeh.plotting.show(fig)
+            return None
+        else:
+            return fig
+
+    @check_types
+    @doc(
+        summary="Plot h12 GWSS data with multiple traces.",
+    )
+    def plot_h12_gwss_multitraces(
+        self,
+        contig: base_params.contig,
+        sample_queries: h12_params.sample_queries,
+        sample_sets: Optional[base_params.sample_sets] = None,
+        colors: gplt_params.colors = bokeh.palettes.d3["Category10"][10],
+        random_seed: base_params.random_seed = 42,
+        title: Optional[gplt_params.title] = None,
+        sizing_mode: gplt_params.sizing_mode = gplt_params.sizing_mode_default,
+        width: gplt_params.width = gplt_params.width_default,
+        track_height: gplt_params.track_height = 170,
+        genes_height: gplt_params.genes_height = gplt_params.genes_height_default,
+        show: gplt_params.show = True,
+        output_backend: gplt_params.output_backend = gplt_params.output_backend_default,
+    ) -> gplt_params.figure:
+        if len(sample_queries) > 0:
+            raise ValueError("At least 1 sample query is required")
+
+        # Plot GWSS track.
+        try:
+            fig1 = self.plot_h12_gwss_track_multi(
+                contig=contig,
+                sample_sets=sample_sets,
+                sample_queries=sample_queries,
+                colors=colors,
+                random_seed=random_seed,
+                title=title,
+                sizing_mode=sizing_mode,
+                width=width,
+                height=track_height,
+                show=False,
+                output_backend=output_backend,
+            )
+
+            fig1.xaxis.visible = False
+            fig1.legend.location = "top_right"
+            fig1.legend.click_policy = "hide"
+
+            # Plot genes.
+            fig2 = self.plot_genes(
+                region=contig,
+                sizing_mode=sizing_mode,
+                width=width,
+                height=genes_height,
+                x_range=fig1.x_range,
+                show=False,
+                output_backend=output_backend,
+            )
+
+            # Combine plots into a single figure.
+            fig = bokeh.layouts.gridplot(
+                [fig1, fig2],
+                ncols=1,
+                toolbar_location="above",
+                merge_tools=True,
+                sizing_mode=sizing_mode,
+            )
+
+            if show:  # pragma: no cover
+                bokeh.plotting.show(fig)
+                return None
+            else:
+                return fig
+        except AssertionError as msg:
+            print(msg)
+            return None
+
+    @check_types
+    @doc(
+        summary="Plot h12 GWSS data with multiple tracks.",
+    )
+    def plot_h12_gwss_multi(
```

Comment:
```suggestion
    def plot_h12_gwss_multi_panel(
```

Maybe make name more descriptive?